src/components/Button/Button.js

```diff
@@ -164,6 +173,7 @@ Button.propTypes = {
 };
 
 Button.defaultProps = {
+  render: null,
```

Comment:
The default prop here seems to have caused some snapshot changes - Probably we can leave it undefined?